src/core/Menu/Menu.tsx

```diff
@@ -17,10 +18,24 @@ export type MenuProps = {
   role?: string;
   /**
    * Menu items. Recommended to use `MenuItem` components.
+   *
+   * If you have custom actionable items, they should have `tabindex={-1}` for better keyboard navigation support
+   * and selected item should have `aria-selected={true}`.
    */
   children: React.ReactNode;
 } & Omit<CommonProps, 'title'>;
 
+const getFocusableItems = (container: HTMLElement | null) => {
+  return [
+    ...getFocusableElements(container),
+    // MenuItem has tabindex="-1".
+    ...Array.from(container?.querySelectorAll('[tabindex="-1"]') ?? []),
```

Comment:
Maybe move this comment in the next line
```suggestion
    ...Array.from(container?.querySelectorAll('[tabindex="-1"]') ?? []), // MenuItem has tabindex="-1".
```

Reply:
Code is gone.